package.json

```diff
@@ -68,9 +68,9 @@
     "babel-standalone": "^6.4.4",
     "codemirror": "^5.12.0",
     "css-loader": "~0.9.0",
-    "react-addons-test-utils": "^0.14.7",
-    "react": "0.14.7",
-    "react-dom": "^0.14.7",
+    "react-addons-test-utils": "^15.0.0 || ^0.14.7",
+    "react": "15.0.1 || ^0.14.7",
+    "react-dom": "^15.0.1 || ^0.14.7",
```

Comment:
Shouldn't all of `react`, `react-dom`, and `react-addons-test-utils` be `^15.0.1 || ^0.14.7`, so they all stay in-step?